SubmittingPatches-kernel.rst

```diff
@@ -0,0 +1,301 @@
+==============================================
```

Comment:
I believe RST does not require markup before and after a line to mark it as a headline.